src/app/routes/api/v3/forms/public-forms.auth.routes.ts

```diff
@@ -22,6 +22,17 @@ PublicFormsAuthRouter.route('/:formId([a-fA-F0-9]{24})/auth/redirect').get(
   PublicFormController.handleFormAuthRedirect,
 )
 
+/**
+ * Removes SP/CP JWT cookie when called to logout user from SP/CP
+ * @route /:authType/logout
+ *
+ * @returns 200 with success message when user logs out successfully
+ * @returns 400 if authType is invalid
+ */
+PublicFormsAuthRouter.route('/:authType/logout').get(
```

Comment:
I was thinking `/auth/:authType/logout` might be better, @karrui wdyt?

Reply:
agree

Reply:
thanks! edited